src/DatePicker/DatePicker.stories.js

```diff
@@ -40,7 +42,7 @@ class DateTimePicker extends React.Component<{}, State> {
     );
     const maxDateStringTimestamp = dateAddon(
       'Max date',
-      new Date('Jan 20 2019')
+      new Date('Feb 20 2019')
```

Comment:
what about changing this to something dynamic base on a current date?
Something like this, but there should be way how to write it nicer then my example.
```
      let today = new Date('Feb 20 2019');
 
      const maxDateStringTimestamp = dateAddon(
        'Max date',	 
        today.setMonth(today.getMonth() + 8);
  )
```

Reply:
Good idea! Implemented. At the end I've used just fixed thirty days offset which is slightly shorter than using `setMonth` `getMonth` combo and the final effect is quite equal.